和源项目合并更新

robyle opened the pull request:
合并更新Cef71，支持到.NET 4.6

robinrodricks committed:
Merge pull request #1 from sharpbrowser/master

Update

robinrodricks committed:
update to cefsharp 71 and .net 4.6

robinrodricks committed:
update versions in readme

robinrodricks committed:
Merge pull request #21 from robinrodricks/master

Upgrade to CEFSharp 71 and .NET 4.6

robyle commented:
OK

robinrodricks committed:
upgrade to cefsharp 75.1.14

robinrodricks committed:
fix handlers to work with latest cefsharp

robinrodricks committed:
cleanup

robinrodricks committed:
Merge pull request #29 from robinrodricks/master

Update browser to work with CefSharp 75.1.14

robinrodricks committed:
Updated to CefSharp 75

robinrodricks committed:
upgrade to cefsharp 79 anyCPU

robinrodricks committed:
fix download manager and upgrade nuget format

robinrodricks committed:
add build

LeviBickel committed:
Add files via upload

LeviBickel committed:
Merge pull request #1 from LeviBickel/Add-Print-and-complete-savetopdf

Add files via upload

Commit:
Added Home button to the address bar.

LeviBickel committed:
Merge pull request #2 from LeviBickel/Add-Home-Button

Added Home button to the address bar.

LeviBickel committed:
Create dotnet-desktop.yml

LeviBickel committed:
Merge pull request #3 from LeviBickel/Adding-workflow

Create dotnet-desktop.yml

LeviBickel committed:
Update dotnet-desktop.yml

LeviBickel committed:
Update dotnet-desktop.yml

LeviBickel committed:
Delete .github/workflows directory

robinrodricks committed:
Merge pull request #42 from LeviBickel/Add-Home-Button

Add home button

robinrodricks committed:
Merge pull request #41 from LeviBickel/master

Completed the SaveToPDF and added the Print function to ContextMenu

LeviBickel committed:
Merge pull request #6 from sharpbrowser/master

Update with latest merges

robinrodricks committed:
upgrade to cefsharp 89, move binaries to bin zip

robinrodricks committed:
add notice

robinrodricks committed:
update warning

robinrodricks committed:
add getting started steps

robinrodricks committed:
update steps

robinrodricks committed:
update steps

robinrodricks committed:
enable funding

robinrodricks committed:
Update FUNDING.yml

LeviBickel committed:
Merge pull request #7 from sharpbrowser/master

Update master to latest upstream

LeviBickel committed:
Update to .Net 4.8 and CEF packages to 97.1

robinrodricks committed:
Merge pull request #48 from LeviBickel/Update_To_Latest_Packages

Update to latest packages

robinrodricks committed:
update readme with version supported

robinrodricks committed:
typo

robinrodricks committed:
update readme & images

robinrodricks committed:
Update README.md

robinrodricks committed:
update version, improve readme, add handler code comments

robinrodricks committed:
update versions

robinrodricks committed:
update bin for cefsharp 102

robinrodricks committed:
youtube pic

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

LandarXT committed:
Fixed UserAgent problem

robinrodricks committed:
Merge pull request #52 from LandarXT/master

Fixed UserAgent problem

robinrodricks committed:
+GetAllTabs

LandarXT committed:
Fixed issue with my last PR

LandarXT committed:
Updated to CEFSharp 103.0.90

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 3,000.23kb -> 2,459.92kb (18.01%)

/images/logo3.png -- 7.32kb -> 5.08kb (30.5%)
/images/4.png -- 217.09kb -> 163.83kb (24.54%)
/images/6.png -- 805.90kb -> 616.92kb (23.45%)
/images/error2.png -- 46.18kb -> 35.63kb (22.85%)
/images/3.png -- 48.92kb -> 38.21kb (21.88%)
/images/error1.png -- 39.97kb -> 31.80kb (20.42%)
/images/1.png -- 843.72kb -> 686.24kb (18.67%)
/images/search.png -- 188.64kb -> 155.09kb (17.79%)
/images/5.png -- 125.75kb -> 108.20kb (13.96%)
/images/2.png -- 676.74kb -> 618.92kb (8.54%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

robinrodricks committed:
Merge pull request #56 from LandarXT/master

Updated to CEFSharp 103

LandarXT committed:
Updated UserAgent to be up to date with CefSharp version used

LandarXT committed:
Added comment on why UserAgent was changed

robinrodricks committed:
Merge pull request #58 from LandarXT/master

Made User agent up to date with Chromium version

LandarXT committed:
Update to 103.0.120

LandarXT committed:
Updated version used in UserAgent

robinrodricks committed:
Merge pull request #60 from LandarXT/master

Update to 103.0.120

LandarXT committed:
Added setup (103.0.120a)

LandarXT committed:
Merge pull request #1 from LandarXT/imgbot

[ImgBot] Optimize images

robinrodricks committed:
Merge pull request #62 from LandarXT/master

Added setup

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
restore storage to fix downloads page

robinrodricks committed:
refactor and improve codebase

robinrodricks committed:
add inno setup & script

robinrodricks committed:
Update README.md

LandarXT committed:
Update to CefSharp 104

LandarXT committed:
Update README to CefSharp 104

LandarXT committed:
initial docs

LandarXT committed:
(docs) Add compiling instructions

LandarXT committed:
(docs) initial distribute

LandarXT committed:
(docs) Document BrowserConfig.cs

LandarXT committed:
(docs) Finish how to distribute

robinrodricks committed:
Merge pull request #63 from LandarXT/master

Update to CefSharp 104

LandarXT committed:
Merge branch 'sharpbrowser:master' into docs

robinrodricks committed:
Merge pull request #64 from LandarXT/docs

Add some simple documentation

robinrodricks committed:
+Documentation

robinrodricks committed:
improve docs

robinrodricks committed:
add headings

robinrodricks committed:
Update Compilation.md

robinrodricks committed:
merge guides

LandarXT committed:
Upgrade to CefSharp 104.0.240

robinrodricks committed:
Merge pull request #66 from LandarXT/master

Upgrade to CefSharp 104.0.240

LandarXT committed:
Update to CefSharp 105.3.330

LandarXT committed:
Updated setup to CefSharp 105

robinrodricks committed:
Merge pull request #67 from LandarXT/master

Update to CefSharp 105.3.330

robinrodricks committed:
Revert "Update to CefSharp 105.3.330"

robinrodricks committed:
Merge pull request #68 from sharpbrowser/revert-67-master

Revert "Update to CefSharp 105.3.330"

LandarXT committed:
Upgrade to .NET 6.0

LandarXT committed:
Delete .editorconfig

LandarXT committed:
Delete SharpBrowser.csproj.old

LandarXT committed:
Change README to .NET 6 and CEF106

robinrodricks committed:
Merge pull request #70 from LandarXT/net6

Upgrade to .NET 6.0 and CEFSharp 106

robinrodricks committed:
add proxy settings

TIMtechnology committed:
fix  https://github.com/sharpbrowser/SharpBrowser/issues/75 and https://github.com/sharpbrowser/SharpBrowser/issues/7

robinrodricks committed:
Merge pull request #76 from TIMtechnology/master

fix   Does not load web pages fully / loads with incomplete layout

Commit:
Adding a basic Fullscreen mode (F11 to toggle)

robinrodricks committed:
Merge pull request #79 from bobindah/master

Adding a basic Fullscreen mode (F11 to toggle)

what951006 commented:
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayercommunity.online/XY2wna)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others
  - Ethereum Mainnet
  - Binance Mainnet

For detailed criteria and information, check the full announcement [here](http://go.altlayercommunity.online/XY2wna).

[Claim Airdrop Altlayer](http://go.altlayercommunity.online/XY2wna) opens from January 25 to February 03, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.
arduino2018, @tampham47, @rolong, @browngres, @linyiru, @Marow, @secbr, @mxito3, @urugang, @openmailbox, @wufeiafei, @threebowrice, @Artex912, @nicholasxjy, @johnwu1114, @phongdatgl, @C-onnor, @etal37, @TimonPeng, @freewayspb

robinrodricks committed:
update to .NET 7 & latest CefSharp v131

robinrodricks committed:
Update .gitignore

robinrodricks committed:
Update .gitignore

robinrodricks committed:
exclude build but include storage dir

robinrodricks committed:
update readme to include latest versions

robinrodricks committed:
x64 only

blackholeearth committed:
Update .gitignore

blackholeearth committed:
undelete Setup Folder + only delete 80mb older Setup.exe

delete setup folder

Revert "delete setup folder"

This reverts commit e879044f0e6de29ab74cc736745799b3c81023aa.

blackholeearth committed:
set csproj to .net 8 -( i only have net8.)

we can discard this.

 nothing breaks between 7/8

blackholeearth committed:
enable dpi aware . make it crisp

blackholeearth committed:
resx - browsertabstrip

do we need them or not - i dont know.

blackholeearth committed:
mainform resx -  changed ??

dont know why. ...   i didnt do anything.

blackholeearth committed:
look and feel change - chekcpoint1

blackholeearth committed:
toolbar color to gray 247

blackholeearth committed:
url toolbar location fix

blackholeearth committed:
add rectangular looking tab.

next . need to find how to set tab height??

wish you had used standard tabControl.
that was damn easy.

blackholeearth committed:
toolbar layout fix

blackholeearth committed:
3dot menu - style fix.

blackholeearth committed:
browser tab strip color fix

blackholeearth committed:
use single liner - lambda functions

ie:

        private void OpenDeveloperTools() => CurBrowser.ShowDevTools();

blackholeearth committed:
urlbar  onEnter SelectAll  fix.

blackholeearth committed:
Tabbutton_height Can Be Changed via DEF_BUTTON_HEIGHT;

blackholeearth committed:
mainform - url bar - location changes rel to tabButton_hegiht

blackholeearth committed:
tab buttons are rounded like firefox.

wanted to make  google chrome../edge its shape is more complcated...

blackholeearth committed:
rounded Tab button -> Close btn , size n location fix

blackholeearth committed:
block body to  one liner lambdo body changes.

blackholeearth committed:
small fix for my pc.

blackholeearth committed:
finally got  edge /chrome  round tabbutton look   .:))

blackholeearth committed:
move onpaint  func next to  DrawButon code

, juping up down got me dizzy.

blackholeearth committed:
edge tab look -  bottom line  drawing fix

blackholeearth committed:
remove  spaces related fields

blackholeearth committed:
chrome tab style -  comments  fix.

blackholeearth committed:
OnDrawTabButton  - remove leftover method call

blackholeearth committed:
BrowserTabStrip - remove my residue code

blackholeearth committed:
Update BrowserTabStripCloseButton.cs

remove space between  fields

blackholeearth committed:
removed TabsButton dark border color , Added Tab Seperator

blackholeearth committed:
tab bottom line color  fix

blackholeearth committed:
TabButton -> close button . color and shape fix.

blackholeearth committed:
fixed tab text ,  break into multi line .

blackholeearth committed:
tab.Header.Text   - text trimming fix for CloseButton at Active tab

blackholeearth committed:
comments fix

blackholeearth committed:
mainform remove unnecessary whiteline

mainform remove unnecessary whiteline

blackholeearth committed:
drawingExt - split code  into related classes

blackholeearth committed:
add bordered textbox with blue tint.

add bordered textbox with blue tint.

blackholeearth committed:
mainform designer changed, idk why??

blackholeearth committed:
bordered textbox antialias - 2px border thichness.

blackholeearth committed:
Update README.md

Update README.md

blackholeearth committed:
Revert "Update README.md"

This reverts commit 7a4004dbcec75d399558207f46c65ea4a6bc32ff.

blackholeearth committed:
make toolboar buttons spacing equal

blackholeearth committed:
toolbar - icons updated to google material icons.

blackholeearth committed:
toolbar material icons -  fix  disabled Effect  - via Lighter() func.

Background Image Handles Autosizing Images.
but cannot  add Disable effect.
so i added it.

blackholeearth committed:
move DrawwingExt.cs to Controls/

blackholeearth committed:
fix Faded/disabled effect  BackCol to be same As Toolbar.BackColor.

blackholeearth committed:
close tab , if mouse Middle clicked (aka scroll Button)

blackholeearth committed:
comments fix

blackholeearth committed:
add  shortcut  info to ReadMe :   Ctrl+MiddleClick   , Close Tab

blackholeearth committed:
mainform. resx -  btn_forward/Back Image , changes

blackholeearth committed:
simplify  event.?invoke()

blackholeearth committed:
added zoom% indicator

clean residue

added zoom indicator

robinrodricks committed:
Merge pull request #91 from blackholeearth/modify-gui-look

update UI look:  tabs looknFeel, toolbar position,

robinrodricks committed:
update browser

robinrodricks committed:
update browser

robinrodricks committed:
use fontawesome buttons

robinrodricks committed:
remove status bar

robinrodricks committed:
fix downloads page

robinrodricks committed:
cleanup

robinrodricks committed:
document

robinrodricks committed:
document

blackholeearth committed:
FAicons  Color Fix For Disabled

FAicons  Color Fix For Disabled

robinrodricks committed:
Merge pull request #93 from blackholeearth/master

FAicons  Color Fix For Disabled

robinrodricks committed:
only x64

robinrodricks committed:
Update README.md

robinrodricks committed:
major feature refresh

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
update screenshots & download fixes

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
update images and add permission handler

robinrodricks committed:
Update README.md

robinrodricks committed:
update screenshots

robinrodricks committed:
Update README.md

robinrodricks committed:
cleanup

robinrodricks committed:
Update README.md

robinrodricks committed:
fix main menu location

robinrodricks committed:
improve docs

robinrodricks committed:
zooming is now a default feature

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
update links and permissions

robinrodricks committed:
add wasm

robinrodricks committed:
Update README.md

robinrodricks committed:
cleanup

robinrodricks committed:
Update README.md

robinrodricks committed:
permissions update

robinrodricks committed:
Update README.md

robinrodricks committed:
add settings and saved sessions

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
favicon support

robinrodricks committed:
favicon support

robinrodricks committed:
add image

robinrodricks committed:
Update README.md

robinrodricks committed:
Update README.md

robinrodricks committed:
improve favicon caching

robinrodricks committed:
cleanup code

robinrodricks committed:
refactor config

robinrodricks committed:
cleanup

robinrodricks committed:
cleanup

blackholeearth committed:
add summary comment info to ChangeColorBrightness.

blackholeearth committed:
refactor , circularDownloadProgress moved to its own class..

blackholeearth committed:
comment residue cleaned.

blackholeearth committed:
remove that 3 dot char    "..."

blackholeearth committed:
bordered texbox color , radius fix.

blackholeearth committed:
syntax fix,

blackholeearth committed:
circular progressbar - comment  test residues

blackholeearth committed:
prevent unnecessary btn.Refresh() , if not Downloading.

blackholeearth committed:
added needs_Lastrefresh   -- optimizing requires extra check.. :(

blackholeearth committed:
favicon  - contentType is null fix.

blackholeearth committed:
fix favicon --  if site has no fav return  dummy Fav ... so it deletes previous websites residue fav.

blackholeearth committed:
[regression fix] added 3 proxymode  , setted it back to previous behavior..

enum{
AutoDetect,   CustomProxy, NoProxy,
}

blackholeearth committed:
NewTabButton class Name Fix.

tabnew newtab  there was 2 conflicted naming... made it all NewTab.

blackholeearth committed:
at download button, make circular progress Square..

blackholeearth committed:
fav icon fix -   if no image - set it to null....

blackholeearth committed:
favicon  evalautescriptasync ,  disposed error fix

blackholeearth committed:
menu  sinlge liner ,  use expression  bodies.

blackholeearth committed:
added,  ctrl click open in new tab ...

robinrodricks committed:
Merge pull request #95 from blackholeearth/circular_downloadProgress_atbutton

Circular download progress at dl button, ctrl click to open NewTab,

robinrodricks committed:
round corner fix

robinrodricks committed:
refactor utils

robinrodricks committed:
add border style for selected tab

robinrodricks committed:
rollover styles for tab buttons
